src/util/struct_array_layout.js.ejs

```diff
@@ -96,4 +96,4 @@ for (const member of members) {
 }
 
 <%=StructArrayLayoutClass%>.prototype.bytesPerElement = <%= size %>;
-register('<%=StructArrayLayoutClass%>', <%=StructArrayLayoutClass%>);
+register('<%=StructArrayLayoutClass%>', <%=StructArrayLayoutClass%>);
```

Comment:
Editors are often configured to automatically add a line break at the end of file (e.g. mine does), so it would be nice to fix this in another place.